Source/WebKit/WebProcess/WebPage/WebCookieCache.cpp

```diff
@@ -65,14 +65,14 @@ static String cookiesToString(const Vector<WebCore::Cookie>& cookies)
     return cookiesBuilder.toString();
 }
 
-String WebCookieCache::cookiesForDOM(const URL& firstParty, const SameSiteInfo& sameSiteInfo, const URL& url, FrameIdentifier frameID, PageIdentifier pageID, IncludeSecureCookies includeSecureCookies)
+String WebCookieCache::cookiesForDOM(const Document& document, const SameSiteInfo& sameSiteInfo, const URL& url, FrameIdentifier frameID, PageIdentifier pageID, IncludeSecureCookies includeSecureCookies)
```

Comment:
Why is the parameter changed? `addChangeListenerWithAccess` only need `firstPartyURL` and it's provided in original function.